src/main/java/uk/gov/hmcts/payment/processors/JobProcessorFactory.java

```diff
@@ -38,6 +38,10 @@ public JobProcessor getJobType(String jobType) {
         if(jobType.equalsIgnoreCase("unprocessed-payment-update")){
             return new UnprocessedPaymentUpdateProcessor();
         }
+
+        if(jobType.equalsIgnoreCase("duplicate-payment-process")){
+            return new DuplicatePaymentProcessor();
+        }
         return null;
     }
```

Comment:
Yes this does cause an exception which is fine because we should never be returning null, probably more elegant to throw an exception with the missing jobType, something for the next update.